ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Your title above should be a short description of what
+is the issue. Do not include the pull request number in the title. -->
+
+#### References
+
+#### Brief description of the issue
+
+#### Reproducible code example
```

Comment:
You might load in the "```python" tics (I used triple quotes that should be back tics below).
```
#### Sample problem
'''python
replace_this_with_your_code
'''
```